libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackItem.java

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.navigation.ui.feedback;
+
+public class FeedbackItem {
+
+  //  @FeedbackEvent.FeedbackType // TODO Telemetry Impl
```

Comment:
Let's cut tickets with this `TODO` and the rest 👇 so we don't forget 👍

Reply:
https://github.com/mapbox/mapbox-navigation-android/issues/2507